STYLE_GUIDE.md

```diff
@@ -0,0 +1,158 @@
+* **Implementation:** Our tools (like `complyctl`) should be modular. Output from one tool should be easily consumable as input for another (e.g., standard JSON/YAML streams).
+
+**2. Convention Over Configuration**
+* **Principle:** Decrease the number of decisions a developer or user needs to make.
+* **Implementation:** Use sensible defaults. Users should only need to specify configuration for things that deviate from the standard norm.
+
+---
+
+## 2. Repository Structure & Standards
+
+Every repository under the ComplyTime organization must contain the following standard files in the root directory to ensure a consistent developer experience:
+
+| File                 | Description                                | Standard                                         |
+|:---------------------|:-------------------------------------------|:-------------------------------------------------|
+| `README.md`          | Project overview, installation, and usage. | Markdown                                         |
+| `LICENSE`            | Legal terms of use.                        | **Apache License 2.0**                           |
+| `CONTRIBUTING.md`    | Guidelines for contributors.               | Link to org-wide guide or repo-specific details. |
+| `CODE_OF_CONDUCT.md` | Community standards.                       | Standard Contributor Covenant                    |
+| `SECURITY.md`        | Security policy.                           | Vulnerability reporting instructions             |
+| `.github/`           | GitHub configuration.                      | Issue templates, PR templates, workflows.        |
+
+> Content of all these files can link to org-wide files, and eventually incremented with repository specific content.
+
+---
+
+## 3. Contribution Workflow
+
+* **Branching Strategy:**
+    * **Main Branch:** `main` is the stable production branch.
+    * **Feature Branches:** Create branches from `main` for all changes.
+* **Pull Requests (PRs):**
+    * **Atomic Changes:** PRs should be small enough to be reviewed in one sitting. Large, sprawling PRs may be requested to be split.
+    * **Review Requirement:** All PRs require review from at least two Maintainers.
+    * **CI/CD Gates:**
+      * **Standard:** All PRs must generally pass automated checks (linting, testing, build) before merging.
+      * **Exceptions:** We recognize that checks may occasionally fail due to external issues outside our control or transient flakes that pass locally. In these rare instances, **maintainers can discuss and agree on exceptions** to merge specific PRs despite a red status.
+    * **Pull Request Title Format**
+      * **Format:** `<type>: <description>`
+      * **Example:** `feat: implement oscal validation logic`
+
+* **Commit Messages:**
+    * Follow the **Conventional Commits** [specification](https://www.conventionalcommits.org/).
+
+---
+
+## 4. Centrally manages configuration files, templates, and standardized CI/CD workflows
```

Comment:
Nit, maybe this title could be a bit smaller since the text below basically says the same. Something like "Infrastructure Standards Centralization"?